query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/order_and_pagination/nested_multi_order_pagination.rs

```diff
@@ -244,22 +239,18 @@ mod paging_one2m_unstable_order {
         // 3 => 5 C C B A
         // 3 => 6 A C D C
         // Makes: [1 => [], 2 => [3, 4] | [4, 3] | [3] | [4], 3 => []]
-        assert_query_many!(
+        insta::assert_snapshot!(
+          run_query!(
             &runner,
             r#"query {
-            findManyTestModel {
-              id
-              related(cursor: { id: 3 }, orderBy: [{ fieldA: desc }, { fieldB: asc }, { fieldC: asc }, { fieldD: desc }]) {
+              findManyTestModel {
                 id
+                related(cursor: { id: 3 }, orderBy: [{ fieldA: desc }, { fieldB: asc }, { fieldC: asc }, { fieldD: desc }]) {
+                  id
+                }
               }
-            }
-          }"#,
-            vec![
-                r#"{"data":{"findManyTestModel":[{"id":1,"related":[]},{"id":2,"related":[{"id":3},{"id":4}]},{"id":3,"related":[]}]}}"#,
```

Comment:
That is somewhat surprising to me that this is different now.

Reply:
There's more examples down below. Curious.

Reply:
Yeah. Once I retested, the results were all consistent.